### PR TITLE
Update loom links

### PR DESCRIPTION
**Pull request by johnnyopao**
updates all references of useloom.com to loom.com

### CODE REVIEW

File: src/sdk/oembed.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { isValidEmbedUrl, isLoomUrl } from './validate';
 
-const LOOM_URL = 'www.useloom.com';
+const LOOM_URL = 'www.loom.com';
```
Comment (vhmth) [inline src/sdk/oembed.js]: @johnnyopao we'll want to intelligently use `www.useloom.com` if the link was a `useloom` link and `www.loom.com` if the link was a `loom` link.

Comment (johnnyopao) [reply src/sdk/oembed.js]: ah gotcha. Will update

Comment (johnnyopao) [reply src/sdk/oembed.js]: done!

**Comment by vhmth** [comment]
Very nice @johnnyopao!

tomfoolerybooth-1BGfFwvmUsvPG5jvIm

**Pull request merged**